src/lib/components/ExternalLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { HTMLProps } from 'react'
+
+/**
+ * Outbound link
+ */
+export default function ExternalLink({
+  target = '_blank',
+  href,
+  rel = 'noopener noreferrer',
+  ...rest
+}: Omit<HTMLProps<HTMLAnchorElement>, 'as' | 'ref' | 'onClick'> & { href: string }) {
```

Comment:
Is there a reason to rewrite the External link we already have? guessing styling stuff

Reply:
This doesn't use GA. We haven't discussed if we want analytics in the widget, but I'm assuming it will be a different arch then just plugging in GA.